app/models/profile.rb

```diff
@@ -2,6 +2,7 @@
 #        :delete for the relationship.  That means no before/after
 #        destroy callbacks will be called on this object.
 class Profile < ApplicationRecord
+  extend ActiveModel::Translation
```

Comment:
It seems to me that this extension isn't needed here

```suggestion
```

Reply:
Yeah, so, there's more than one way to make this kind of thing work with `I18n`. 

@fdocr's suggestion goes at this by attaching the message to `:base` (which won't attempt to translate any attribute name) with the full message coming from `validators.profile_validator.bio_too_long`. That works, it's a good approach. 

I had suggested this `ActiveModel::Translation` module, combined with adding a key in `config/locales/models/*.yml` like this:

```yaml
en:
  activerecord:
    attributes:
      profile:
        summary: Bio
```

You could probably also use `alias_attribute`, but that approach feels awkward to me.

I guess it's not clear to me if this attribute is mis-named (eg, we really should call it "bio" but we don't want to migrate, etc.) or if the name is specific to the error message / validator context. (Like, do we call it "Bio" everywhere now? Or do we maybe we call it "Summary" sometimes and "Bio" other times?)

All/any of these are viable approaches, though, I'm fine with whatever works.